src/system/on-dhcp-event.sh

```diff
@@ -1,12 +1,20 @@
 #!/bin/bash
-
-# This script came from ubnt.com forum user "bradd" in the following post
-# http://community.ubnt.com/t5/EdgeMAX/Automatic-DNS-resolution-of-DHCP-client-names/td-p/651311
-# It has been modified by Ubiquiti to update the /etc/host file
-# instead of adding to the CLI.
-# Thanks to forum user "itsmarcos" for bug fix & improvements
-# Thanks to forum user "ruudboon" for multiple domain fix
-# Thanks to forum user "chibby85" for expire patch and static-mapping
+#
+# Copyright (C) 2024 VyOS maintainers and contributors
```

Comment:
just a nit, but I don't think it is technically legal to take a contribution of one explicitly credited person and replace it with your own copyright unless they explicitly signed it off while copying their work verbatim.

Reply:
The source code from that script doesn't exist here anymore.